quasar/main.tf

```diff
@@ -286,12 +286,12 @@ resource "aws_db_parameter_group" "quasar-prod" {
     value = "2000000"
   }
 
-  # Recommended by PGTuner tool: https://pgtune.leopard.in.ua/#/
+  # Updated to match Heroku Premium-5 config which we experienced
```

Comment:
@sheyd The comment refers to Premium-5, but on the [Heroku Postgres docs](https://devcenter.heroku.com/articles/heroku-postgres-plans#premium-tier) I see 120 MB associated with the Premium-6 tier. Am I missing something?

Reply:
@mshmsh5000 Do you mean 122GB RAM size? I'm not seeing the param stats published on the page you linked, and I did a `show run` on a Premium-5 Heroku DB earlier this morning, and the `work_mem` Postgres parameter was `120MB` 🤔